server.js

```diff
@@ -0,0 +1,58 @@
+const net = require('net');
+const files = require('./files.js');
+
+const PORT = 8080;
+let date = new Date();
+date = date.toUTCString();
```

Comment:
Don't set Date here since it'll have the same date for all requests going into the future and not the current date of the request itself.